src/profile/travelocity/extension/tab-type.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+    if (localStorage.getItem(window.location.href)) {
```

Comment:
Tony,

We need the whole URL for this extension. As per the requirement, if the already opened web page (for eg- Search Results or Infosite page) is opened again in a new tab , it should be treated as the same session, rather than a new one. So we need to have the full URL with paramaters to check whether it is the specific page itself.  Car SR page URL -https://www.travelocity.com/carsearch?dagv=1&subm=1&fdrp=&styp=4&locn=Las%20Vegas%2C%20Nevada&date1=08/18/2017&date2=08/19/2017&vend=&kind=1&time1=1030AM&time2=1030AM&ttyp=2&acop=2&rdus=10&rdct=1

Here by checking till "?" will not be sufficient to identify whether it is the same CAR SR page because till "?" URL would be same even if we try for CAR Search for London location.